src/OctoshiftCLI.Tests/gei/Commands/MigrateRepoCommandTests.cs

```diff
@@ -849,7 +860,49 @@ public async Task It_Falls_Back_To_Github_Target_Pat_If_Github_Source_Pat_Is_Not
                 GITHUB_TARGET_PAT,
                 GITHUB_TARGET_PAT,
                 It.IsAny<string>(),
-                It.IsAny<string>()));
+                It.IsAny<string>(),
+                It.IsAny<bool>()));
+        }
+
+        [Fact]
+        public async Task It_Skips_Releases_When_Option_Is_True()
+        {
+            // Arrange
+            var environmentVariableProviderMock = new Mock<EnvironmentVariableProvider>(null);
+
+            var mockGithub = new Mock<GithubApi>(null, null, null);
```

Comment:
```suggestion
            var mockGithub = TestHelpers.CreateMock<GithubApi>();
```

Reply:
Updated in 3ddc2374b698be4f5b2e1a721c85fed6cddf2ec6!